lib/stm32wba/hci/ll_sys_intf.c

```diff
@@ -21,6 +21,9 @@
 #include "event_manager.h"
 #include "ll_intf.h"
 
+extern uint8_t AHB5_SwitchedOff;
+extern uint32_t radio_sleep_timer_val;
```

Comment:
This can't work. You're not supposed to use Zephyr code here `uint32_t/uint8_t`.
This should be Cube code only.